src/parseJsonReports.ts

```diff
@@ -5,30 +5,38 @@ import { JsonFinal } from './types/JsonFinal';
 import { JsonSummary } from './types/JsonSummary';
 import { stripIndent } from 'common-tags';
 
-const parseJsonReport = async <type extends JsonSummary | JsonFinal>(jsonSummaryPath: string): Promise<type> => {
-  const resolvedJsonSummaryPath = path.resolve(process.cwd(), jsonSummaryPath);
+/** Parse a Vitest coverage reporter file */
```

Comment:
Do you think those comments are really needed?
Maybe we an avoid them with a better name for the function? (like `parseVitestCoverageReport`)?

Reply:
Honestly, this is probably just a stylistic preference; I almost always try to have some basic summary comments on functions, and added them by default. I do like your suggestion of renaming the functions though instead 💯.